DNN Platform/DotNetNuke.Abstractions/Logging/IEventLogConfigService.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace DotNetNuke.Abstractions.Logging
+{
+    using System.Collections;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// The event log config service provides APIs for managing the
+    /// Event Log configuration and individual logs itself.
+    /// </summary>
+    public interface IEventLogConfigService
+    {
+        /// <summary>
+        /// Adds the log type.
+        /// </summary>
+        /// <param name="configFile">The configuration file.</param>
+        /// <param name="fallbackConfigFile">The fallback configuration file.</param>
+        void AddLogType(string configFile, string fallbackConfigFile);
+
+        /// <summary>
+        /// Adds the log type config info.
+        /// </summary>
+        /// <param name="logTypeConfig">The log type configuration.</param>
+        void AddLogTypeConfigInfo(ILogTypeConfigInfo logTypeConfig);
+
+        /// <summary>
+        /// Adds the log.
+        /// </summary>
+        /// <param name="logType">The log type.</param>
+        void AddLogType(ILogTypeInfo logType);
+
+        /// <summary>
+        /// Clears the log.
+        /// </summary>
+        void ClearLog();
+
+        /// <summary>
+        /// Deletes the log.
+        /// </summary>
+        /// <param name="logInfo">the log info.</param>
+        void DeleteLog(ILogInfo logInfo);
```

Comment:
This one doesn't seem like it's config related either.  Does this fit better on `IEventLogger`?

Reply:
In the purest sense of a logger, you should never be able to delete a log statement only add log statements. I think it may make sense to have a 3rd interface such as `IEventLogDeleteService`. This would allow us to have all the delete APIs and clear APIs

Reply:
This has been updated in the latest code push, thanks for the feedback